package.json

```
@@ -3,9 +3,12 @@
   "version": "0.11.6",
```

lets bump the version, and you will want to run `$ npm install` and `$ npm install leaflet --no-save` again to regenerate the package.lock with the new version number too
```suggestion
   "version": "0.12.6",
```

---

:+1: